lib/index.js

```diff
@@ -16,12 +17,13 @@ function colorize(string, hexColor) {
     })];
 }
 
+function _fetch(url) {
+  return fetch(url, {headers: headers})
```

Comment:
(as this is modules requires at least node 5, this could be shortened to `{ headers }` in this case)